packages/editor-ui/src/components/ParameterInputFull.vue

```diff
@@ -28,14 +28,14 @@
 			>
 				<template v-slot="{ droppable, activeDrop }">
 					<n8n-tooltip
-						placement="bottom-start"
+						placement="left"
 						:manual="true"
 						:value="showMappingTooltip"
 					>
-						<div
-							slot="content"
-							v-html="$locale.baseText(`dataMapping.${displayMode}Hint`, { interpolate: { name: parameter.displayName } })"
-						/>
+						<div :class="$style.tooltipContent" slot="content">
+							<span v-html="$locale.baseText(`dataMapping.${displayMode}Hint`, { interpolate: { name: parameter.displayName } })" />
+							<n8n-button :class="$style.dismiss" type="primary" :label="$locale.baseText('_reusableBaseText.dismiss')" @click="onMappingTooltipDismissed"/>
```

Comment:
as we had discussed with Max, would be nice if this was a global pattern for tooltips.. because Max intends to reuse this..

Reply:
ok, I'll do this in a new PR